1.preprocessing_data/README.md

```diff
@@ -2,52 +2,36 @@
 
 In this module, we present our pipeline for preprocessing the CFReT pilot data.
 
-## Illumination Correction
+## Illumination Correction (IC)
 
-To correct for illumination issues within the CFReT pilot data, we use the BaSiC method that was established in an article by [Peng et al.](https://doi.org/10.1038/ncomms14836).
-We specifically use the Python implementation of this method, called [PyBaSiC](https://github.com/peng-lab/BaSiCPy).
```

Comment:
I noticed there were some updated PyBaSiC functions within `utils`. As a double check here, is PyBaSiC still used?

Reply:
That was me testing with PyBaSiC to get it to work. Some of the changes helped but since it isn't reliable, we don't use it anymore. The utils file has been moved to a git ignored folder to avoid confusion.